lxc/image.go

```diff
@@ -256,7 +256,14 @@ func (c *cmdImageCopy) Run(cmd *cobra.Command, args []string) error {
 		imgInfo.Fingerprint = name
 	}
 
+	newAliases := make([]api.ImageAlias, len(c.flagAliases))
```

Comment:
Im not sure why we need this bit, please can you explain?

Reply:
An other misunderstanding... I thought we needed to check that the copy was creating a target alias, which is not necessarily the case. I removed it.